server/app/services/export/JsonExporter.java

```diff
@@ -132,12 +132,16 @@ private void exportToJsonApplication(
   }
 
   public static void exportEntriesToJsonApplication(
-      CfJsonDocumentContext jsonApplication, ImmutableMap<Path, ?> entries) {
-
-    for (Map.Entry<Path, ?> entry : entries.entrySet()) {
+      CfJsonDocumentContext jsonApplication, ImmutableMap<Path, Optional<?>> entries) {
```

Comment:
Another solution to this would be to have the QuestionJsonPresenters return a DocumentContext that we insert at the specified path. Then the JsonExporter doesn't have to care what the types are.

That'd be a larger refactor, though.